webui/src/Views/InstanceView.elm

```diff
@@ -45,6 +45,14 @@ jobControlsColumnWidth =
     200
 
 
+allocationStateColumnWidth =
+    190
+
+
+allocationLogColumnWidth =
+    110
```

Comment:
If a column has fixed content I prefer to have the width fixed. This way the layout is more consistent across multiple of those tables.